docs/rest_api_guide/05_RESTdeveloper.rst

```diff
@@ -0,0 +1,132 @@
+Developer API Calls
+===================
+
+A number of calls exist in the API that are targeted at those developing
+pipelines and plugins for Fledge. These are not actions that are expected
+to be of everyday use, but are to aid in this development process.
+
+Purge Readings
+--------------
+
+Under ordinary circumstances a user should never need to manually purge
+data from the Fledge storage buffer, however during the development
+process it can be useful to be able to manually purge data.
+
+``DELETE /fledge/asset`` - Purge data for all assets from the buffer
+
+**Response Payload**
+
+The response payload is a JSON document that returns the number of readings that have been deleted.
+
+**Example**
+
+.. code-block:: console
+
+   $ curl -X DELETE http://localhost:8081/fledge/asset
+
```

Comment:
As we say above Response payload : we might need to add JSON response how it looks like.

```
{"purged": XXX}
```